src/Elm/Parser/Tokens.elm

```diff
@@ -197,17 +196,17 @@ multiLineStringLiteral : Core.Parser String
 multiLineStringLiteral =
     Core.succeed identity
         |. Core.symbol "\"\"\""
-        |= Core.loop [] multiLineStringLiteralStep
+        |= Core.loop "" multiLineStringLiteralStep
 
 
-multiLineStringLiteralStep : List String -> Core.Parser (Step (List String) String)
-multiLineStringLiteralStep partsSoFarReverse =
+multiLineStringLiteralStep : String -> Core.Parser (Step String String)
+multiLineStringLiteralStep stringSoFar =
     Core.oneOf
         [ Core.symbol "\"\"\""
-            |> Core.map (\() -> Done (List.Extra.listReverseThenStringConcatFast partsSoFarReverse))
+            |> Core.map (\() -> Done stringSoFar)
         , Core.symbol "\""
-            |> Core.mapChompedString (\v () -> Loop (v :: partsSoFarReverse))
-        , Core.succeed (\v -> Loop (String.fromChar v :: partsSoFarReverse))
+            |> Core.mapChompedString (\v () -> Loop (stringSoFar ++ v))
```

Comment:
`Core.map (\() -> Loop (stringSoFar ++ "\"") `  should be even faster, right?

(PS: I'm just mentioning the improvements I think of, you stop and mark this as ready for review whenever you want :D )